File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/entitiesvb/vb/createupdateentitymetadata2.vb
' <snippetcreateupdateentitymetadata2>


 Dim createBankNameAttributeRequest As CreateAttributeRequest = New CreateAttributeRequest With {
  .EntityName = _customEntityName,
  .Attribute = New StringAttributeMetadata With {
   .SchemaName = "new_bankname",
   .RequiredLevel = New AttributeRequiredLevelManagedProperty(AttributeRequiredLevel.None),
   .MaxLength = 100,
   .Format = StringFormat.Text,
   .DisplayName = New Label("Bank Name", 1033),
   .Description = New Label("The name of the bank.", 1033)
  }
 }

 _serviceProxy.Execute(createBankNameAttributeRequest)

' </snippetcreateupdateentitymetadata2>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/entities/cs/createupdateentitymetadata2.cs
// <snippetcreateupdateentitymetadata2>


 CreateAttributeRequest createBankNameAttributeRequest = new CreateAttributeRequest
 {
  EntityName = _customEntityName,
  Attribute = new StringAttributeMetadata
  {
   SchemaName = "new_bankname",
   RequiredLevel = new AttributeRequiredLevelManagedProperty(AttributeRequiredLevel.None),
   MaxLength = 100,
   FormatName = StringFormatName.Text,
   DisplayName = new Label("Bank Name", 1033),
   Description = new Label("The name of the bank.", 1033)
  }
 };

 _serviceProxy.Execute(createBankNameAttributeRequest);

// </snippetcreateupdateentitymetadata2>